team/onboarding.rst

```diff
@@ -43,16 +43,10 @@ Data Facility resources
 
 You'll also need an account for the LSST Data Facility services, hosted at NCSA, which include:
 
-- Development Servers (`lsst-dev </services/lsst-dev.html>`_)
-- MySQL Database (`lsst-dev-db </services/lsst-db.html>`_)
+- Development Servers (`lsst-dev </services/lsst-login.html>`_)
```

Comment:
At least for the time being, I think it would be appropriate to mention the ``lsst-dev`` systems here: I understand from @mbutlerncsa that they are not being retired quite yet, and not all of the services that they currently provide are yet available elsewhere.

Reply:
This talks about accounts. These are the same accounts. Listing both implies different accounts to me.
We point new users to lsst-login and provide information about lsst-dev and other resources

Reply:
Actually the new documentation does say lsst-dev, but points to the documentation that was moved under lsst-login. That might be confusing, but is probably less confusing for newish developers that may have been told that they need an 'lsst-dev' account. I think we leave it as revised here.